pool/src/lib.rs

```diff
@@ -658,10 +724,12 @@ impl LendingPool {
     fn require_good_position(
         env: &Env,
         who: Address,
+        who_balance: Option<i128>,
```

Comment:
I recommend to use `mb_` or `may_be` prefix for `Option` arguments and variables